### DISCUSSION TITLE
Finding BIC value for fitted model

### DISCUSSION THREAD

**Discussion by yildizyasin** [CamDavidsonPilon/lifelines#1274] [Q&A] [answered]
There is an attribute returns AIC  value for fitted model (`FITTER.AIC_`) but I couldn't find it for BIC. Is there a simple way to find BIC (_Bayesian Information Criterion_) value for fitter model? 

```
exf = ExponentialFitter().fit(T, E, label="ExponentialFitter")
print(exf.AIC_) # works

print(exf.BIC_) # needed

```

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @yildizyasin, it's not implemented. Not for any particular reason though. It would be a good addition.

**Reply by yildizyasin** [reply]
It is already implemented in find_best_method part. However, not sure if this calculation is correct. Or, am I missing something?
Here is a formula for BIC
https://machinelearningmastery.com/probabilistic-model-selection-measures/

_The BIC statistic is calculated for logistic regression as follows (taken from “The Elements of Statistical Learning“):_

**BIC = -2 * LL + log(N) * k**
_Where log() has the base-e called the natural logarithm, LL is the log-likelihood of the model, N is the number of examples in the training dataset, and k is the number of parameters in the model._

I can implement it if we get aligned on this formula.


Now:
```

"BIC": lambda model: 2 * len(model._fitted_parameter_names)
- 2 * model.log_likelihood_ * np.log(model.event_observed.shape[0]),

```

Then:

```
"BIC_new": lambda model: -2 * model.log_likelihood_
        +  len(model._fitted_parameter_names) * np.log(model.event_observed.shape[0]),

```

**Reply by CamDavidsonPilon** [reply]
Yup, looks like I mixed up the terms. How silly! Can you write a PR to fix it?

**Reply by yildizyasin** [reply]
Just created pull request, [#1275](https://github.com/CamDavidsonPilon/lifelines/pull/1275)
